Regression/WarpX-tests.ini

```
@@ -2656,322 +2632,281 @@ useOMP = 1
 dim = 2
-addToCompileString = USE_PSATD=TRUE
-cmakeSetupOpts = -DWarpX_DIMS=2 -DWarpX_PSATD=ON
+addToCompileString =
+cmakeSetupOpts = -DWarpX_DIMS=2
 restartTest = 0
 useMPI = 1
-numprocs = 1
+numprocs = 2
 useOMP = 1
 numthreads = 1
 compileTest = 0
 doVis = 0
-compareParticles = 1
-particleTypes = electrons ions
-analysisRoutine = Examples/Tests/nci_psatd_stability/analysis_galilean.py
+analysisRoutine = Examples/Tests/pml/analysis_pml_ckc.py
 
-[galilean_2d_psatd_current_correction]
+[pml_x_psatd]
 buildDir = .
-inputFile = Examples/Tests/nci_psatd_stability/inputs_2d
-runtime_params = psatd.periodic_single_box_fft=0 psatd.update_with_rho=0 psatd.current_correction=1 diag1.fields_to_plot=Ex Ey Ez Bx By Bz jx jy jz rho divE amr.max_grid_size=64 amr.blocking_factor=64
+inputFile = Examples/Tests/pml/inputs_2d
+runtime_params = algo.maxwell_solver=psatd psatd.update_with_rho=1 warpx.do_dynamic_scheduling=0 diag1.fields_to_plot = Ex Ey Ez Bx By Bz rho divE warpx.cfl = 0.7071067811865475 warpx.do_pml_dive_cleaning=0 warpx.do_pml_divb_cleaning=0 chk.file_prefix=pml_x_psatd_chk chk.file_min_digits=5 psatd.current_correction=0 warpx.abort_on_warning_threshold=medium
 dim = 2
 addToCompileString = USE_PSATD=TRUE
 cmakeSetupOpts = -DWarpX_DIMS=2 -DWarpX_PSATD=ON
-restartTest = 0
+restartTest = 1
+restartFileNum = 150
 useMPI = 1
 numprocs = 2
 useOMP = 1
 numthreads = 1
 compileTest = 0
 doVis = 0
-compareParticles = 1
-particleTypes = electrons ions
-analysisRoutine = Examples/Tests/nci_psatd_stability/analysis_galilean.py
+analysisRoutine = Examples/Tests/pml/analysis_pml_psatd.py
 
-[galilean_2d_psatd_hybrid]
+[pml_x_yee]
 buildDir = .
-inputFile = Examples/Tests/nci_psatd_stability/inputs_2d_hybrid
-runtime_params = psatd.current_correction=0 warpx.abort_on_warning_threshold=medium
+inputFile = Examples/Tests/pml/inputs_2d
+runtime_params = warpx.do_dynamic_scheduling=0 algo.maxwell_solver=yee chk.file_prefix=pml_x_yee_chk chk.file_min_digits=5
 dim = 2
-addToCompileString = USE_PSATD=TRUE
-cmakeSetupOpts = -DWarpX_DIMS=2 -DWarpX_PSATD=ON
-restartTest = 0
+addToCompileString =
+cmakeSetupOpts = -DWarpX_DIMS=2
+restartTest = 1
+restartFileNum = 150
 useMPI = 1
 numprocs = 2
 useOMP = 1
 numthreads = 1
 compileTest = 0
 doVis = 0
-compareParticles = 1
-particleTypes = electrons ions beam
-analysisRoutine = Examples/analysis_default_regression.py
+analysisRoutine = Examples/Tests/pml/analysis_pml_yee.py
 
-[comoving_2d_psatd_hybrid]
+[pml_x_yee_eb]
 buildDir = .
-inputFile = Examples/Tests/comoving/inputs_2d_hybrid
-runtime_params = psatd.current_correction=0 warpx.abort_on_warning_threshold=medium
+inputFile = Examples/Tests/pml/inputs_2d
+runtime_params = warpx.do_dynamic_scheduling=0 algo.maxwell_solver=yee chk.file_prefix=pml_x_yee_eb_chk chk.file_min_digits=5
 dim = 2
-addToCompileString = USE_PSATD=TRUE
-cmakeSetupOpts = -DWarpX_DIMS=2 -DWarpX_PSATD=ON
-restartTest = 0
+addToCompileString =
```

This isn't really related to ordering the tests, but I noticed that the `pml_x_yee_eb` test runs with the 2D Cartesian tests due to this missing line. This forces a recompile in the 2D tests which adds several minutes to an already pretty long test, so moving it to the appropriate (Embedded Boundary) set of tests would be good.
```suggestion
addToCompileString = USE_EB=TRUE
```

---

good catch !